components/json_page/index.coffee

```diff
@@ -30,13 +30,13 @@ module.exports = class JSONPage
         .end (err, res) =>
```

Comment:
It's that we're handling 'res.error' instead or the 'err' object returned in the callback.